CMakeLists.txt

```diff
@@ -13,6 +13,39 @@ option(ENABLE_AUDIO "Enable Audio" ON)
 option(ENABLE_MICROPROFILE "Enable microprofile" OFF)
 option(ENABLE_ANGELSCRIPT "Enable AngelScript" ON)
 option(ENABLE_MOFILEREADER "Enable MofileReader" ON)
+option(CROSSBUILD_EMSCRIPTEN "Build for the EmScripten Platform" OFF)
+
+if (CROSSBUILD_EMSCRIPTEN)
+  # Turn off not features that are not compatible with emscripten
+
+  include(~/.conan/data/emsdk_installer/1.39.6/bincrafters/stable/package/1492a59deb6efdf29776a5734de63fc5f5c58650/upstream/emscripten/cmake/Modules/Platform/Emscripten.cmake )
```

Comment:
I'm not sure this is good practice...

Reply:
The hash is likely to change... We should use the proper conan way to do this

Reply:
You're right about both, but i don't know how to do this with conan. Maybe @AnotherFoxGuy could help us out here?